index.html

```diff
@@ -58,5 +61,227 @@ <h3 id="connect">LET’S CONNECT</h3>
+        <div class="dot"><img src="/images/doticon.png"></div>
+        <div class="tonic-p"><p>Back End Dev</p></div>
+        <div class="dot"><img src="/images/doticon.png"></div>
+        <div class="tonic-p"><p>2015</p></div>
+      </div>
+      <div class="project-p">
+        <p>
+          A daily selection of privately personalized reads; 
+          no accounts or sign-ups required. 
+        </p>
+      </div>
+      <div class="skills">
+        <ul>
+          <li>
+            html
+          </li>
+          <li>
+            css
+          </li>
+          <li>
+            javascript
+          </li>
+        </ul>
+      </div>
+      <nav class="see">
+        <button class="see-project">
+        <a class="project-btn">See Project</a>
+        </button>
+      </nav>
+    </div>
+  </section>
+  <section class="about">
+    <div class="abou-last">
+      <section class="bio2">
+        <header class="bio2">
+          <h1 id="title2">About <br> Myself</h1>
+          <p id="paragraph2">Hello I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! If you like what 
+            you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+          <h3 id="connect2">LET’S CONNECT</h3>
+      </header>
+      </section>
+      <nav class="bottom-icons">
+        <ul>
+          <li>
+            <a href="#"><img src="/images/github.png"></a>
+          </li>
+          <li>
+            <a href="#"><img src="/images/Linkedin icon.png"></a>
+          </li>
+          <li>
+            <a href="#"><img src="/images/hand.png"></a>
+          </li>
+          <li>
+            <a href="#"><img src="/images/twitter.png" ></a>
+          </li>
+          <li>
+            <a href="#"><img src="/images/m.png"></a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <nav class="resume">
+      <button class="resume-btn">
+      <a class="get-btn">Get my resume</a>
+      </button>
+    </nav>
+    <div class="arrows" id="first-arrow">
+      <h2>Languages</h2>
+      <img clas="arrowimg" src="/images/Normal Button/Tertiary/Icons/downarrow.png">
+    </div>
+      <div class="languages">
+        <div class="languages-content">
+          <img src="/images/js.png">
+          <p>JavaScript</p>
+        </div>
+        <div class="languages-content">
+          <img src="/images/html.png">
+          <p>JavaScript</p>
+        </div>
+        <div class="languages-content">
+          <img src="/images/css.png">
+          <p>JavaScript</p>
+        </div>
+      </div>
+    <div class="arrows" id="arrow-top">
+      <h2>FrameWorks</h2>
+      <img clas="arrowimg" src="images/Normal Button/Tertiary/Icons/normarrow.png">
+    </div>
+    <div class="arrows" id="arrow-box">
+      <h2>Skills</h2>
+      <img clas="arrowimg" src="/images/Normal Button/Tertiary/Icons/normarrow.png">
```

Comment:
- [ ] Kindly make sure that the Skills'  narrow is viewed in the browser for a better user interface. 👇🏽 
![image](https://user-images.githubusercontent.com/58642949/192077746-0cccf5b0-165b-4781-a938-f09691d1119d.png)